tabs/advanced_tuning.js

```diff
@@ -124,9 +107,39 @@ TABS.advanced_tuning.initialize = function (callback) {
         $('#advanced-tuning-save-button').click(function () {
             saveChainer.execute();
         });
-
+        
+        $('a.save').click(function () {
+            Settings.saveInputs().then(function () {
+                var self = this;
+                MSP.promise(MSPCodes.MSP_EEPROM_WRITE);
+                GUI.log('OSD settings saved');
```

Comment:
Please, change this log message to something more relevant, or alternatively, just remove it entirely.

Reply:
yea. just remove it.